### PR TITLE
correct command for using specified openssl with cmake

### PR DESCRIPTION
**Pull request by ByteYue**
# Pull Request Checklist

Hi there, I'm trying to integrate the Azure SDK into [apache doris](https://github.com/apache/doris), and during the process, I found that I need to specify a particular version of OpenSSL. At this point, I discovered that the related commands in the documentation were incorrect, so I'm attempting to correct the documentation through this PR.

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

**Review by LarryOsterman — APPROVED** [review]

**Comment by LarryOsterman** [comment]
Thank you very much for catching my typo, the change looks good.

Let me know if you're done with verifying the fixes so I can merge the changes in.

File: README.md [hunk 1]
```diff
@@ -268,7 +268,7 @@ vcpkg install --overlay-ports=<path to the vcpkg-custom-ports above>
 If you are building using CMAKE, you can instruct CMAKE to apply the overlay ports using the following command line switches:
 
 ```sh
-vcpkg -DVCPKG_MANIFEST_MODE=ON -DVCPKG_OVERLAY_PORTS=<path to the vcpkg-custom-ports above> -DVCPKG_MANIFEST_DIR=<path to the directory containing the vcpkg.json file>
+cmake -DVCPKG_MANIFEST_MODE=ON -DVCPKG_OVERLAY_PORTS=<path to the vcpkg-custom-ports above> -DVCPKG_MANIFEST_DIR=<path to the directory containing the vcpkg.json file>
```
Comment (ahsonkhan) [inline README.md:271]: I was initially having trouble finding docs for `VCPKG_MANIFEST_MODE`
I didn't see `VCPKG_MANIFEST_MODE` in the vcpkg Environment variable docs:
https://learn.microsoft.com/en-us/vcpkg/users/config-environment

Here's the cmake integration page lists these variables:
https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration

**Review by ahsonkhan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ahsonkhan) [reply README.md:271]: @LarryOsterman or @ByteYue please feel free to resolve this conversation, before merging.

**Review by antkmsft — APPROVED** [review]

**Comment by ByteYue** [comment]
> Thank you very much for catching my typo, the change looks good.
> 
> Let me know if you're done with verifying the fixes so I can merge the changes in.

Hi @LarryOsterman , sorry for my late response. I have already passed the verification, everything is fine now, and it’s ready to be merged.

**Pull request merged**